packages/core/src/libraries/hook.ts

```diff
@@ -90,7 +95,7 @@ export const createHookLibrary = (queries: Queries) => {
           .post(url, {
             headers: { 'user-agent': 'Logto (https://logto.io)', ...headers },
             json,
-            retry: { limit: retries },
+            retry: { limit: hookRetryLimit },
```

Comment:
just put 3 here?

Reply:
since we also have the timeout number here..

Reply:
Updated.